client/pages/userAccount.jsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import defaultStates from '../lib/defaultStateCount';
+import * as topojson from 'topojson-client';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+const d3 = window.d3;
+
+export default class UserAccount extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      accountId: 1,
+      isLoading: true
+    };
+    this.renderInforgraphic = this.renderInforgraphic.bind(this);
+    this.infographicMap = React.createRef();
+  }
+
+  componentDidMount() {
+    const { accountId } = this.state;
+    fetch(`/api/accounts/${accountId}`)
+      .then(response => response.json())
+      .then(result => {
+        if (result[0].length !== 0) {
+          result[0].forEach(element => {
+            const stateCode = element.stateCode;
+            if (defaultStates[stateCode]) {
+              defaultStates[stateCode].visits = element.visits;
+            }
+          });
+          this.renderInforgraphic();
+          this.setState({
+            results: result[0],
+            total: result[1][0].reviews
+          });
+        } else {
+          this.renderInforgraphic();
+          this.setState({
+            results: null,
+            total: 'N/A'
+          });
+        }
+      })
+      .catch(err => console.error(err));
+  }
+
+  renderInforgraphic() {
+    const dataObject = {};
+    for (const key in defaultStates) {
+      const stateName = defaultStates[key].name;
+      const visits = defaultStates[key].visits;
+      dataObject[stateName] = Number(visits);
+    }
+    const color = d3.scaleQuantize()
+      .domain([1, 15])
+      .range(d3.schemeGreens[9]);
+
+    const path = d3.geoPath();
+    const svg = d3.select('#map')
+      .append('svg')
+      .attr('viewBox', '0 0 975 610');
+
+    const toolTip = d3.select('#map')
+      .append('div')
+      .style('position', 'absolute')
+      .attr('class', 'tooltip');
+
```

Comment:
In React apps we want to avoid querying the DOM when possible. Fortunately we've got a ref

```js
const svg = d3.select(this.infographicMap.current)

```